modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/DeployableArtifactDescriptorFactoryTestCase.java

```diff
@@ -371,11 +377,80 @@ public void classLoaderConfigurationWithPluginDependencyAndAdditionalDependencie
   @Test
   public void classLoaderConfigurationWithPluginDependencyAndAdditionalDependenciesLightweightUseLocalRepository()
       throws Exception {
+    replacePlaceholderInClassloaderModel();
     final String location = "/plugin-dependency-with-additional-dependencies-lightweight-local-repository";
     populateRepositoryDependencies(new File(getArtifact(getArtifactRootFolder() + location), "local-repository"));
     assertClassLoaderConfigurationWithPluginDependencyAndAdditionalDependencies(location);
   }
 
+  private void replacePlaceholderInClassloaderModel() throws IOException {
```

Comment:
can't be pass this to the non abstract class. For example creating a protected method that returns "apps" by default in ApplicationDescriptorFactoryTestCase and that is overridden in DomainDescriptorFactoryTestCase with "domains" and that is set in the root.